### PR TITLE
Issue 1810 more tests for quoting

### PR DESCRIPTION
**Pull request by michael-volynets**
Issue #1810 

Time Spent: ~4h

In general, went without issues. but there were small things I didn't know. (e.g. I didn't know that the iframe composer src path from 'flowcrypt-consumer' is different from 'flowcrypt-consumer-mock' (even if both messages are present in both consumers))

### CODE REVIEW

**Comment by michael-volynets** [comment]
I something went wrong with CI? I didn't make any changes there

**Comment by tomholub** [comment]
Yeah, not your fault. I had to increase the instance size for tests.

**Comment by tomholub** [comment]
Basically, our browsers were crashing during ci tests, sometimes. I starts about 15 browsers at the same time.

Increasing ram from 4gb to 8gb seems to help. Let's see.

**Comment by michael-volynets** [comment]
Okay, got it.
Thanks

**Comment by tomholub** [comment]
> but there were small things I didn't know. (e.g. I didn't know that the iframe composer src path from 'flowcrypt-consumer' is different from 'flowcrypt-consumer-mock' (even if both messages are present in both consumers))

That's interesting. I would think the url should be exactly the same. If it doesn't work with the url exactly the same, maybe that's a bug that should be fixed.

**Comment by tomholub** [comment]
I bug in the mock api, maybe.

**Comment by michael-volynets** [comment]
Hmm, I can double check it. Maybe, I pasted src incorrectly

File: extension/js/common/composer.ts [hunk 1]
```diff
@@ -1919,7 +1919,14 @@ export class Composer {
       this.S.cached('icon_show_prev_msg').show().addClass('progress');
       Xss.sanitizeAppend(this.S.cached('icon_show_prev_msg'), '<div id="loader">0%</div>');
       this.resizeComposeBox();
-      this.messageToReplyOrForward = await this.getAndDecryptPreviousMessage((progress) => this.setQuoteLoaderProgress(progress + '%'));
+      try {
+        this.messageToReplyOrForward = await this.getAndDecryptPreviousMessage((progress) => this.setQuoteLoaderProgress(progress + '%'));
+      } catch (e) {
+        Catch.reportErr(e);
```
Comment (tomholub) [inline extension/js/common/composer.ts]: Here you can do:
```js
if(Api.err.isSignificant(e)) {
  Catch.reportErr(e);
}
```
That way it will only report errors that are not server 5xx, network errors and similar (eg stuff that we should actually fix).

**Comment by tomholub** [comment]
You don't have to double check it now. But next time you see that, you can share a sample that works and sample that doesn't etc, I can take a look.

File: extension/js/common/composer.ts [hunk 2]
```diff
@@ -1919,7 +1919,14 @@ export class Composer {
...
+        await Ui.modal.error(`The error occured while getting ${method === 'reply' ? 'replied' : 'forwarded'} message. The messasge won't be included in new message.`);
```
Comment (tomholub) [inline extension/js/common/composer.ts]: Here at the end you can add `\n${Api.err.eli5(e)}`

eli5 = "Explain like I'm 5". It will add user-friendly explanation of what exactly went wrong.

**Comment by michael-volynets** [comment]
Okay, got it

File: extension/js/common/composer.ts (hunk 1 above)
Comment (michael-volynets) [reply extension/js/common/composer.ts]: Yeah, I forgot to add this. Thanks!

File: extension/js/common/composer.ts (hunk 2 above)
Comment (michael-volynets) [reply extension/js/common/composer.ts]: aha, I was thinking about why it's called `eli5` 
Now, I know :)
I've added it

**Comment by tomholub** [comment]
Looks good

**Comment by michael-volynets** [comment]
I don't know if you remove this line accidentally or not 
```TS
        this.S.cached('icon_show_prev_msg').remove();
```

but in the composer, there will be loader with 0% loaded. I don't think it's a good way to show it.
image

**Comment by michael-volynets** [comment]
Or you wanted to try to reload the previous message by clicking on it?

**Comment by tomholub** [comment]
Sorry, that needs a fix. I wanted to reset the loader so that it would be the same as if user never clicked it. Then they can click the three dots again to try again.

**Comment by tomholub** [comment]
Thanks for checking
